tests/timeseries/mock_resampler.py

```diff
@@ -187,58 +188,65 @@ async def _handle_resampling_requests(self) -> None:
                 )
             )
 
+    def make_sample(self, value: float | None) -> Sample[Quantity]:
+        """Create a sample with the given value."""
```

Comment:
`Args` and `Returns` are not mandatory for helpers in tests, right?

Reply:
The current configuration doesn't require to document arguments or returns when the docstring is one line only, in all the code. @shsms bought this up recently so we are going to talk about it in the next meeting.